pkgs/top-level/all-packages.nix

```diff
@@ -40231,4 +40231,6 @@ with pkgs;
   zf = callPackage ../tools/misc/zf { };
 
   isolate = callPackage ../tools/security/isolate { };
+
+  qc = callPackage ../development/tools/qc { };
```

Comment:
The `all-packages.nix` are roughly sorted by category and alphabetical order of package names.

We can find `../development/tools/qtcreator/` in line 18806,  we should put `qc` before it

Reply:
Thank you, I moved it.